src/components/DetailsView/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useContext } from 'react';
+import ModiteContext from '../../state/modite';
+import Modite from '../../models/Modite';
+import classNames from 'classnames/bind';
+import s from './styles.module.css';
+import ModiteProfileResp from '../../models/ModiteProfileResp';
+
+// TODO: type correctly
+function Details({ className = '' } : any) {
+  const [activeModite, setActiveModite]: [Modite, React.Dispatch<any>] = useContext(ModiteContext);
```

Comment:
Watch for unused vars, `setActiveModite` isn't used here. We could set `no-unused-variable` tslint rule to also catch this.

Reply:
fixed